docs/en/installing.rst

```diff
@@ -122,31 +133,19 @@ Create a new project and select one of the available boards. You can change afte
    board = esp32-c3-devkitm-1
    framework = arduino
 
-How to update to the latest code
-********************************
+How to update to the stable code based on 3.0.x 
+-----------------------------------------------
 
-To test the latest Arduino ESP32, you need to change your project *platformio.ini* accordingly.
-The following configuration uses the upstream version of the Espressif development platform and the latest Arduino core directly from the Espressif GitHub repository:
+To use the actual Arduino ESP32, you need to change your project *platformio.ini* accordingly.
+The following configuration uses the community version pioarduino and the latest Arduino core from the pioarduino GitHub repository:
 
 .. code-block:: bash
 
    [env:esp32-c3-devkitm-1]
-   platform = https://github.com/platformio/platform-espressif32.git
+   platform = https://github.com/pioarduino/platform-espressif32/releases/download/51.03.03/platform-espressif32.zip
```

Comment:
Maybe use the link to the latest version ?

```suggestion
   platform = https://github.com/pioarduino/platform-espressif32/releases/latest/download/platform-espressif32.zip
```

Reply:
Honestly, I don't like. By doing this you don't know what version is used.
When a new release is done, the project updates automatically to the latest version (maybe incompatible to the project).

Reply:
Good idea about the version as an example

Reply:
Maybe you can add something like (feel free to improve it):

```
.. note:: Note that this is only an example. Check the latest available version in `pioarduino's repository <https://github.com/pioarduino/platform-espressif32/releases>`_.
```

Reply:
added. Thx!